h_transport_materials/references.bib

```diff
@@ -2316,3 +2316,54 @@ @article{malo_membranes_2022
     abstract = {Deuterium permeation through vanadium membranes in a wide range of pressures and the temperature range ~250&ndash;550 &deg;C was experimentally investigated. Measurements on the same material were carried out in three laboratories with different features for an extended characterization and for cross-check validation. A unified equation for deuterium permeability in pure vanadium (99%) was provided as &Phi;=1.27&times;10&minus;4&middot;e&minus;8667/T mol m&minus;1 s&minus;1 Pa&minus;0.5, which represents a significant progress for the characterization of the transport properties in this material, given the spread of data, which can currently be found in the literature. Adsorption and recombination rate constants were also measured for hydrogen and deuterium at low pressure for the same range of temperatures. Finally, the influence of the surface roughness was examined by measuring samples with different surface finish.},
     doi = {10.3390/membranes12060579}
 }
+
+@article{HU_2015,
```

Comment:
If you don't mind  could we conform to the [author]_[first word of paper]_[year] standard for entry names?

For instance for this it would be hu_hydrogen_2015